docs/release/new-release-series.md

```diff
@@ -33,85 +37,76 @@ Do first, anytime before the month of the release
 Do afterward, anytime before the month of the release
 -----------------------------------------------------
 
--   Create a blank `osg-3.4` SVN branch and add `buildsys-macros` package
-
-    1.  svn copy `buildsys-macros` from trunk and hand-edit it to hardcode all the new values
-
-        For EL 6:
+-   Create a blank `osg-3.X` SVN branch and add `buildsys-macros` package
 
-        1.  Edit the spec file and set `dver` to 6
-        2.  Run the following commands (adjust the NVR as necessary):
-
-                :::console
-                $ osg-build rpmbuild --el6
-                $ osg-koji import _build_results/buildsys-macros-*.el6.src.rpm
-                $ osg-koji import _build_results/buildsys-macros-*.el6.noarch.rpm
-                $ osg-koji tag-pkg osg-3.4-el6-development buildsys-macros-7-5.osg34.el6
+    1.  svn copy `buildsys-macros` from trunk and hand-edit it to hardcode the new `osg_version` and `dver` values
 
-        Then do the same for EL 7:
+        Do the following steps for all EL versions relevant to the new series; eg, for EL 7:
 
-        1.  Edit the spec file and set `dver` to 7
+        1.  Edit the spec file and set `dver` to `7`, and `osg_version` to `3.X`
         2.  Run the following commands (adjust the NVR as necessary):
 
                 :::console
                 $ osg-build rpmbuild --el7
                 $ osg-koji import _build_results/buildsys-macros-*.el7.src.rpm
                 $ osg-koji import _build_results/buildsys-macros-*.el7.noarch.rpm
-                $ osg-koji tag-pkg osg-3.4-el7-development buildsys-macros-7-5.osg34.el7
+                $ pkg=$(basename _build_results/buildsys-macros-*.el7.src.rpm)
+                $ pkg=${pkg%.src.rpm}
+                $ osg-koji tag-pkg osg-3.X-el7-development "$pkg"
 
     2.  Do a 'real' build of `buildsys-macros`
 
         1.  Bump the revision in the `buildsys-macros` spec file and edit the `%changelog`.
-            **Again, you will need a version of osg-build with 3.4 support.**
+            **Again, you will need a version of osg-build with 3.X support.**
 
-        2.  Set `dver` to 6. Commit
-
-                :::console
-                $ osg-build koji --repo=3.4 --el6
+        Do the following steps for all EL versions relevant to the new series; eg, for EL 7:
 
         3.  Set `dver` to 7. Commit
 
                 :::console
-                $ osg-build koji --repo=3.4 --el7
+                $ osg-build koji --repo=3.X --el7
 
 - Update `tarball-client` scripts
     - `bundles.ini`
     - `patches/`
-    - `upload-tarballs-to-oasis` (for 3.4, `foreach_dver_arch` will need to be updated because we're dropping i386 support)
+    - `upload-tarballs-to-oasis` (for 3.X, `foreach_dver_arch` will need to be updated for the new set of 3.X `dver_arches`)
```

Comment:
```suggestion
    - `upload-tarballs-to-oasis` (`foreach_dver_arch` may need to be updated for the new set of 3.X `dver_arches`)
```

Reply:
The upload-tarballs-to-oasis script defines one set of `dver_arches` per 3.x release series, so it always needs to be updated.

Reply:
OK